app/flatpak-builtins-list-remotes.c

```diff
@@ -194,10 +150,10 @@ gboolean
 flatpak_complete_list_remotes (FlatpakCompletion *completion)
 {
   g_autoptr(GOptionContext) context = NULL;
-  g_autoptr(FlatpakDir) dir = NULL;
 
   context = g_option_context_new ("");
-  if (!flatpak_option_context_parse (context, options, &completion->argc, &completion->argv, 0, &dir, NULL, NULL))
+  if (!flatpak_option_context_parse (context, options, &completion->argc, &completion->argv,
+                                     FLATPAK_BUILTIN_FLAG_ONE_DIR, NULL, NULL, NULL))
```

Comment:
Flags should be 0 here, no?

Reply:
yeah, not sure what I was thinking

Reply:
Now that I'm looking at it again I remember; it was that I was trying to maintain that the first commit didn't introduce any functional changes. So I'll move that change into a separate commit.